x-pack/plugins/ml/public/application/management/jobs_list/components/insufficient_license_page.tsx

```diff
@@ -5,78 +5,61 @@
   basePath: CoreStart['http']['basePath'];
 }
 
 export const InsufficientLicensePage: FC<Props> = ({ basePath }) => (
-  <Fragment>
-    <EuiPage data-test-subj="mlPageInsufficientLicense">
-      <EuiPageBody>
-        <EuiPageContentHeader>
-          <EuiPageContentHeaderSection>
-            <EuiTitle>
-              <h1>
-                <FormattedMessage
-                  id="xpack.ml.management.jobsList.insufficientLicenseTitle"
-                  defaultMessage="Machine Learning"
-                />
-              </h1>
-            </EuiTitle>
-          </EuiPageContentHeaderSection>
-        </EuiPageContentHeader>
-        <EuiPageContentBody>
-          <EuiSpacer size="m" />
-          <EuiCallOut
-            title={i18n.translate('xpack.ml.management.jobsList.insufficientLicenseLabel', {
-              defaultMessage:
-                'Machine Learning is only available on a trial, platinum or enterprise license',
-            })}
-            color="danger"
-            iconType="cross"
-          >
-            <EuiText size="s">
-              <p>
-                <FormattedMessage
-                  id="xpack.ml.management.jobsList.insufficientLicenseDescription"
-                  defaultMessage="Please {link} to use Machine Learning features"
-                  values={{
-                    link: (
-                      <EuiLink
-                        href={`${basePath.get()}/app/management/stack/license_management/home`}
-                      >
-                        <FormattedMessage
-                          id="xpack.ml.management.jobsList.insufficientLicenseDescription.link"
-                          defaultMessage="upgrade your license or start a trial"
-                        />
-                      </EuiLink>
-                    ),
-                  }}
-                />
-              </p>
-            </EuiText>
-          </EuiCallOut>
-        </EuiPageContentBody>
-      </EuiPageBody>
-    </EuiPage>
-  </Fragment>
+  <>
+    <EuiPageHeader
+      data-test-subj="mlPageInsufficientLicense"
+      pageTitle={
+        <FormattedMessage
+          id="xpack.ml.management.jobsList.insufficientLicenseTitle"
+          defaultMessage="Machine Learning"
+        />
+      }
+      bottomBorder
+    />
+
+    <EuiSpacer size="l" />
+    <EuiPageContent verticalPosition="center" horizontalPosition="center" color="danger">
+      <EuiEmptyPrompt
+        iconType="alert"
+        title={
+          <h2>
+            <FormattedMessage
+              id="xpack.ml.management.jobsList.insufficientLicenseLabel"
+              defaultMessage="Insufficient license"
+            />
+          </h2>
+        }
+        body={
+          <p>
+            <FormattedMessage
+              id="xpack.ml.management.jobsList.insufficientLicenseDescription"
+              defaultMessage="Machine Learning is only available on a trial, platinum or enterprise license. Please {link} to use Machine Learning features"
```

Comment:
```suggestion
              defaultMessage="Machine Learning is available only with trial, platinum or enterprise licenses. Please {link} to use Machine Learning features"
```

Reply:
Done in https://github.com/elastic/kibana/pull/103117/commits/8caf4d90e9754cf4c83e834f9a799642bbd77c50